frame/dapps-staking/src/lib.rs

```diff
@@ -59,40 +87,29 @@ impl Default for Forcing {
 
 /// A record for total rewards and total amount staked for an era
 #[derive(PartialEq, Eq, Clone, Default, Encode, Decode, RuntimeDebug, TypeInfo)]
-pub struct EraRewardAndStake<Balance: HasCompact> {
+pub struct EraInfo<Balance: HasCompact> {
     /// Total amount of rewards for an era
     #[codec(compact)]
     pub rewards: Balance,
-    /// Total staked amount for an era
+    /// Total staked amount in an era
     #[codec(compact)]
     pub staked: Balance,
+    /// Total locked amount in an era
+    #[codec(compact)]
+    pub locked: Balance,
 }
 
 /// Used to split total EraPayout among contracts.
 /// Each tuple (contract, era) has this structure.
 /// This will be used to reward contracts developer and his stakers.
-#[derive(Clone, PartialEq, Encode, Decode, RuntimeDebug, TypeInfo)]
-pub struct EraStakingPoints<AccountId: Ord, Balance: HasCompact> {
+#[derive(Clone, PartialEq, Encode, Decode, Default, RuntimeDebug, TypeInfo)]
+pub struct EraStakingPoints<Balance: HasCompact> {
```

Comment:
I agree and don't like this name at all.
It's remnant from the legacy.

Personally, I'd like to get rid of *Points* entirely since there are none in the struct.

Perhaps `DappStakingInfo` or `ContractStakingInfo`?

Reply:
So, may be we could drop `Staking` from the name, whats about `DappInfo`?

Reply:
We already have that one [here](https://github.com/AstarNetwork/astar-frame/pull/17/files/b06c0c514da703841618eea521477aee3d1a2ae7#diff-74f8acb4bf207fcaa637a96dcba0e2d302253ac759392b2799bf2f5a32938745R52-R58).

Reply:
Any other idea maybe?

I'm for `ContractStakeInfo`.